.gitignore

```diff
@@ -1,2 +1,8 @@
 .coverage
 coverage.lcov
+bowtie2.1.bt2
```

Comment:
you might want to be more flexible (as you don't know how many parts will be generated) by using wildcards, i.e. `bowtie2.*.bt2`